bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/device/MessageHandler.java

```diff
@@ -381,6 +381,29 @@ public void handleMessage(int group, byte cmd1, Msg msg,
 		}
 	}
 
+	public static class RampOnOffSwitchHandler extends MessageHandler {
+		RampOnOffSwitchHandler(DeviceFeature p) { super(p); }
```

Comment:
How would this work with other devices that use 0x2E for ramp on and 0x2F for ramp off? see: http://www.insteon.com/developer/#devdocs.